### DISCUSSION TITLE
Job summaries don't appear to be available

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#45895] [Mobile] [answered]
GitHub Mobile beta

Go to a workflow summary

https://github.com/check-spelling/cassandra/actions/runs/4053939447

![Screenshot_20230131-073508.png](https://user-images.githubusercontent.com/2119212/215764176-e1021239-43d2-4cf4-b003-cf6cc0b808c6.png)

Chrome mobile view

![Screenshot_20230131-074908.png](https://user-images.githubusercontent.com/2119212/215764397-c0a54d86-8bc1-4678-a31a-6307b00d318a.png)

**Comment by alcere** [comment, marked as answer]
Hi @jsoref, thanks for your feedback, I will pass the feedback to the team, and they will evaluate the request for future iterations.

**Reply by jsoref** [reply]
Fwiw, some actions support was added recently:
- https://github.com/community/community/discussions/54943
